src/containers/Login/component.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import {
+  Button, Col, Form, Input, Layout, Row, Typography, Alert
+} from 'antd'
+import { UserOutlined, LockOutlined } from '@ant-design/icons'
+import PropTypes from 'prop-types'
+
+const Login = ({
+  login, errorTxt, loading
+}) => (
+  <div className="center">
+    <Layout>
+      <Layout.Content>
+        <Row
+          type="flex"
+          justify="center"
+        >
+          <Col
+            sm={{ span: 10, offset: 1 }}
+            lg={{ span: 8, offset: 2 }}
+          >
+            <Form onFinish={login}>
+              <Typography.Title>The Movie DB</Typography.Title>
+              {errorTxt && (
+              <Alert
+                type="error"
+                message={errorTxt}
+              />
+              )}
+              <Form.Item
+                name="username"
+              >
```

Comment:
you don't need to add new line after component name if you plan to pass only one property